pkgs/by-name/sh/shader-slang/package.nix

```
@@ -76,8 +80,10 @@ stdenv.mkDerivation (finalAttrs: {
       miniz
       lz4
       libxml2
-      spirv-headers
     ]
+    ++ (lib.optionals devendorSpirvHeaders [
+      spirv-headers
+    ])
     ++ (lib.optionals stdenv.hostPlatform.isLinux [
       libX11
     ])
```

```suggestion
    ++ lib.optionals devendorSpirvHeaders [
      spirv-headers
    ]
    ++ lib.optionals stdenv.hostPlatform.isLinux [
      libX11
    ]
```
that is unnecessary code

---

Yeah I'm kinda surprised that nixfmt didn't remove it